src/App.tsx

```diff
@@ -2,13 +2,14 @@ import React from "react";
 import "./App.css";
 import Sidebar from "./components/Sidebar";
 import Feed from "./components/Feed";
-import Post from "./components/Post";
+import Widget from "./components/Widget";
 
-function App() {
+function App(): JSX.Element {
```

Comment:
Please use React.FC as return type.

JSX.Element is used when returning snippets under a larger react component

Reply:
Thanks for pointing out the issue. Corrected